cmd/stack_validate_build.go

```diff
@@ -0,0 +1,76 @@
+// Copyright © 2019 IBM Corporation and others.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"strings"
+	"time"
+)
+
+// Simple test for appsody build command. A future enhancement would be to verify the image that gets built.
+func TestBuild(projectDir string) error {
+
+	// appsody build
+
+	imageName := "testbuildimage"
+
+	Info.Log("******************************************")
+	Info.Log("Running appsody build")
+	Info.Log("******************************************")
+	_, err := RunAppsodyCmdExec([]string{"build", "--tag", imageName}, projectDir)
+	if err != nil {
+		Error.Log(err)
+		return err
+	}
+
+	// It will take a while for the image to build, so lets use docker image ls to wait for it
+	fmt.Println("calling docker image ls to wait for the image")
```

Comment:
Looks to me like the `appsody build` command runs synchronously, so do we really need this code to "wait" for the build to finish?

A `docker image ls` is still good to make sure the image was created. What other tests could we do on the image? Maybe a docker run? This can be a future PR.